src/components/data/SelectionDrawer.js

```diff
@@ -40,10 +47,23 @@ function SelectionToolbar(props) {
         onClose,
         acceptedType,
         multiSelect,
+        getToolbarHeight,
     } = props;
+    const theme = useTheme();
     const classes = useStyles();
     const [displayPath, setDisplayPath] = useState("");
 
+    const toolbarRef = useRef(null);
+
+    // useEffect will run on toolbarRef value assignment
+    useEffect(() => {
+        // The 'current' property contains info of the reference:
+        // align, title, ... , width, height, etc.
+        if (toolbarRef.current && getToolbarHeight) {
+            getToolbarHeight(toolbarRef.current.offsetHeight);
+        }
+    }, [toolbarRef, getToolbarHeight]);
```

Comment:
And finally, this `useEffect` could be replaced with `setToolbarRef(toolbarRef)`